src/components/Obligations.vue

```diff
@@ -14,11 +14,13 @@
         <h3>{{ requirement.title }}</h3>
         <p>{{ requirement.elements[score[3] - 1].text }}</p>
       </div>
-      <p class="row">
-        <a :href="$t('linkDirective', locale)" target="_blank">
+      <div class="row">
+        <h3>{{ $t("otherRequirementsTitle", locale) }}</h3>
+        <p>{{ $t("otherRequirements", locale) }}</p>
```

Comment:
Not sure if you just want to wrap the whole text in the link or just pull merge the 2 paragraphs together

Reply:
Looks good like this I think.
![image](https://user-images.githubusercontent.com/53863781/110039032-9ce1f400-7d0e-11eb-99ed-d740cd54a0aa.png)